api/src/main/java/org/motechproject/nms/api/web/UserController.java

```diff
@@ -38,10 +41,17 @@
 @Controller
 public class UserController extends BaseController {
 
+    public static final String CALLING_NUMBER = "callingNumber";
+    public static final String LANGUAGE_LOCATION_CODE = "languageLocationCode";
+    public static final String SERVICE_NAME = "serviceName";
```

Comment:
Are these the type of constants that we envision putting in the config/props package or is that just magic numbers and such?